diff --git a/core/Makefile b/core/Makefile
index 2e0bc11..47c6d7c 100644
--- a/core/Makefile
+++ b/core/Makefile
@@ -283,7 +283,7 @@ DEFAULT_KEY_CERT_PAIR := $(DEFAULT_SYSTEM_DEV_CERTIFICATE)
 # Rules that need to be present for the all targets, even
 # if they don't do anything.
 .PHONY: systemimage
-systemimage:
+systemimage:kernelheader
 
 # -----------------------------------------------------------------
 
